CONTRIBUTING.md

```diff
@@ -138,3 +138,26 @@ When a new version of dap is to be released, you _must_ follow the instructions
 2. Edit [lib/dap/version.rb](https://github.com/rapid7/dap/blob/master/lib/dap/version.rb) and increment ```VERSION```.  Commit and push to rapid7/dap master.
 3. Run `rake release`.  Among other things, this creates the new gem, uploads it to Rubygems and tags the release with a tag like `v<VERSION>`, where `<VERSION>` is replaced with the version from `version.rb`.  For example, if you release version 1.2.3 of the gem, the tag will be `v1.2.3`.
 4. If your default remote repository is not `rapid7/dap`, you must ensure that the tags created in the previous step are also pushed to the right location(s).  For example, if `origin` is your fork of dap and `upstream` is `rapid7/master`, you should run `git push --tags --dry-run upstream` to confirm what tags will be pushed and then `git push --tags upstream` to push the tags.
+
+## Misc tips on building dap
+
+Ruby often comes prepackaged on linux/mac os systems. Although the README already mentions using rbenv, it useful to make sure your envoiroment is actually using the rbenv version of ruby, gem, & bundler running ```bundle install```:
+
+```
```

Comment:
I like the idea of this section but suggest that instead be done as plain text, otherwise it raises more questions than it answers.  For example, what if rbenv or gem is in a slightly different path?  What then?    See my comment on the previous paragraph.